schema/va-spec/base/va-core-source.yaml

```
@@ -1012,6 +1012,61 @@ $defs:
     - focusAlleleFrequency
     - cohort
 
+  TumorVariantFrequencyStudyResult:
+    inherits: StudyResult
+    maturity: trial use
```

```suggestion
    maturity: draft
```

This was just ideated the other day; let's put it at draft for now and see how this plays out on a few different datasets.